### PR TITLE
Bedrock updates and basic example

### PR DESCRIPTION
**Pull request by jamie256**
* Minor fixes and cleanup
* Add calls to WhyLabsSecureAPI for bedrock
* Add separate example notebooks for Bedrock Titan express and OpenAI
* Update version info

### CODE REVIEW

File: notebooks/Instrumenting_Bedrock.ipynb
Comment (FelipeAdachi) [inline notebooks/Instrumenting_Bedrock.ipynb:1]: it'd be nice to have a screenshot of the traces on the platform at the end of this notebook, but we can do that in a follow-up

Comment (jamie256) [reply notebooks/Instrumenting_Bedrock.ipynb:1]: I can add that in a follow up.

File: openllmtelemetry/instrumentation/bedrock/__init__.py [hunk 1]
```diff
@@ -48,6 +48,43 @@ def wrapper(wrapped, instance, args, kwargs):
 
     return _with_tracer
 
+def _handle_request(secure_api: Optional[WhyLabsSecureApi], span, prompt: str):
```
Comment (FelipeAdachi) [inline openllmtelemetry/instrumentation/bedrock/__init__.py]: in _handle_request, span comes before prompt, but in _handle_response, span comes after prompt/response. We could set the same param order for both functions.

**Review by FelipeAdachi — APPROVED** [review]
LGTM!

**Pull request merged**